dojo.py

```diff
@@ -1,569 +1,480 @@
-                    Dojo.app_session['room'].append(room)
-                    print(Dojo.app_session['room'])
-                    print("office room was created")
-                    rooms.append(room)
-                else:
-                    print('room already exists can you name it something else')
-
-        elif room_type.lower() == valid_rooms[1]:
-            for index, room_name in enumerate(room_names):
-                room_data = {}
-                if not (room_exists(room_name, 'room')):
-                    room = Room('living', room_name)
-                    # key_name = room_name
-                    # room_name = create_room(room_name, 'living')
-                    # print(room_name.room_name)
-                    # room_data['room_name'] = room_name.room_name
-                    # room_data['room_type'] = room_name.room_type
-                    # room_data['room_occupants'] = room_name.room_occupants
-                    # room_data['max_occupants'] = room_name.maxoccupant
-                    # room_data['room_size'] = room_name.add_room_size()
-                    Dojo.app_session['room'].append(room)
-                    rooms.append(room)
-                    print(Dojo.app_session['room'])
-                    print("living room was created")
->>>>>>> ladi
-                else:
-                    print('{0} already exists can you name it something else'.format(room_name))
-
-
-        else:
-            raise TypeError
-        return rooms
-
-    def add_person(self, fname, lname, person_type, wants_accommodation='n'):
-        valid_persons = ['fellow', 'staff']
-        if (person_type.lower() == 'fellow') and (wants_accommodation.lower() == 'y' or wants_accommodation.lower()== 'yes'):
-            person_data = {}
-            person_name = '{0} {1}'.format(fname, lname)
-            if (unique_name(person_name, 'person')):
-                key_name = person_name
-                person_name = create_person(person_name, 'fellow', 'y')
-                person_data['person_name'] = key_name
-                person_data['person_type'] = 'fellow'
-                room_name = add_person_to_room(key_name, 'n')
-                person_data['person_office'] = room_name
-                living_name = add_person_to_room(key_name, 'y')
-                person_data['person_living'] = living_name
-                app_session['person'].append(person_data)
-                print('Fellow {0} has been successfully added'.format(key_name))
-                if room_name.strip() != '':
-                    print ('{0} has been allocated the office {1}'.format(fname, room_name))
-                if room_name.strip() != '':
-                    print ('{0} has been allocated a livingspace {1}'.format(fname, living_name))
+        self.app_session = {'room': {}, 'person': {}}
+        self.all_rooms = []
+        self.people = []
+        self.people_keys = []
```

Comment:
we don't require this as we can get the same from the list of people instances..